xbmc/utils/FileExtensionProvider.cpp

```diff
@@ -123,6 +145,28 @@ void CFileExtensionProvider::SetAddonExtensions(const TYPE& type)
   m_addonExtensions.insert(make_pair(type, StringUtils::Join(extensions, "|")));
 }
 
+void CFileExtensionProvider::SetAddonFileFolderExtensions(const TYPE& type)
```

Comment:
Thought we could combine both in `SetAddonExtensions` and add the conditions there, but its fine too.

Reply:
sure. different rules in different projects, my colleagues would hate on me for doing two things in one method.